test/adapter/json_api/resource_type_config_test.rb

```diff
@@ -35,6 +35,11 @@ def with_jsonapi_resource_type type
             ActiveModel::Serializer.config.jsonapi_resource_type = old_type
           end
 
+          ProfileTypeSerializer = Class.new(ActiveModel::Serializer) do
+            attributes :name
+            type 'profile'
+          end
+
```

Comment:
Let's do regular classes plz :)  I don't know what the previous ones were like this.

```ruby
class ResourceTypeConfigTest < Minitest::Test
  class ProfileTypeSerializer < ActiveModel::Serializer
    attributes :name
    type 'profile'
   end
```